CMakeLists.txt

```diff
@@ -75,6 +76,60 @@ if (GLOW_WITH_OPENCL)
   find_package(OpenCL REQUIRED)
 endif ()
 
+# Returns the name of the backend enabling
+MACRO(getBackendEnableVariable result backend_dir_name)
```

Comment:
we try to keep this CMake lean, can you plase these macros to the cmake/modules/GlowDefaults

Reply:
Done. I've moved the external backend related cmake code in cmake/modules